resources/faq.md

```diff
@@ -179,11 +179,11 @@ Until our first stable release (v1.0), MeiliSearch minor versions are not compat
 
 ## What are the recommended requirements for hosting a MeiliSearch instance?
 
-**The short answer: we don't know.**
+**The short answer: we don't know yet!**
 
-The complexity involved in the indexing process makes it extremely difficult to accurately estimate the size and memory use of a MeiliSearch database, even knowing the size of the `.JSON` files beforehand.
+Indexing documents is a highly complex process, making it difficult to accurately estimate the size and memory use of a MeiliSearch database. In addition, our engine is improving almost constantly.
```

Comment:
Isn't `almost constantly` a borderline contradiction in terms? If the problem is that some updates actually cause performance regressions (and thus we're not *improving* constantly), perhaps we can do something like:

```suggestion
Indexing documents is a highly complex process, making it difficult to accurately estimate the size and memory use of a MeiliSearch database. At this stage, our engine is changing constantly, which means benchmarks are constantly out of date.
```

Reply:
Nice change, thanks! 😄